client/my-sites/domains/domain-management/edit/card/header/primary-domain-button.jsx

```diff
@@ -27,7 +27,7 @@ class PrimaryDomainButton extends React.Component {
 		if ( domain && ! domain.isPrimary ) {
 			return (
 				<Button compact onClick={ this.handleClick }>
-					{ translate( 'Make Primary' ) }
+					{ translate( 'Make primary' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.